src/plugins/azgenerator/CodeModelAz.ts

```diff
@@ -181,8 +181,10 @@ export interface CodeModelAz {
     GetPreparerEntities(): any[];
     GatherInternalResource();
     FindExampleWaitById(id: string): string[][];
+    GetExampleItems(example: CommandExample, isTest: boolean, commandParams: any): string[];
```

Comment:
I am wondering when do we call this GetExampleItems and pass isTest as true?

Reply:
exists: https://github.com/Azure/autorest.az/blob/master/src/plugins/azgenerator/CodeModelAzImpl.ts#L1985

Reply:
Got it. thx